test/e2e/misc_test.go

```diff
@@ -7,14 +7,15 @@ import (
 	"crypto/tls"
 	"encoding/json"
 	"fmt"
-	"github.com/jaegertracing/jaeger-operator/pkg/util"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/jaegertracing/jaeger-operator/pkg/util"
```

Comment:
This import looks misplaced.

Reply:
@jpkrohling Where should it be?  Neither make format nor make lint complain about it.

Reply:
utils.go seems to be the same - but possibly try moving it to the bottom group of imports (line 28) and see if it complains.